### planner environment settings ##############################################

# the size of the used hash map (the bigger the better); should be something
# with 2^X (initially 2^16=65536)
max_hash_size: 65536

# the heuristic that should be used to estimate the step costs of a planning 
# state possible choices: 
# EuclideanHeuristic, EuclStepCostHeuristic, PathCostHeuristic
heuristic_type: PathCostHeuristic


### planner settings ###########################################################

# define the planner of choice provided by the SBPL package
# - ARAPlanner
# - ADPlanner
# - RSTARPlanner
planner_type: ARAPlanner

# search until a specific time limit is reached or first solution is found
#search_until_first_solution: False
search_until_first_solution: False

# the maximum search time if search_until_first_solution is set to False

allocated_time: 2.0
# allocated_time: 5.0

initial_epsilon: 10.0

forward_search: False

# the limit of changed states that decides whether to replan or to start a hole
# new planning task
changed_cells_limit: 20000000
